package.json

```diff
@@ -129,6 +130,7 @@
     "react-i18next": "^11.5.0",
     "react-localization": "1.0.13",
     "react-native": "0.63.3",
+    "react-native-async-storage": "^0.0.1",
```

Comment:
I guess u installed it by mistake. Remove it and reinstall `node_modules` to update `package-lock.json`

Reply:
done